tools/internal/ocpaths/ocpaths.go

```diff
@@ -173,28 +165,50 @@ func insert(dstMap map[OCPathKey]*OCPath, src *OCPath) error {
 	return nil
 }
 
+func convertOCPath(ocpathProto *ppb.OCPath) *OCPath {
+	return &OCPath{
+		Key: OCPathKey{
+			Path:      ocpathProto.GetName(),
+			Component: ocpathProto.GetOcpathConstraint().GetPlatformType(),
```

Comment:
Would it be more clear if this were `PlatformType` instead of `Component`?

Reply:
Done. PTAL